packages/app/src/app/overmind/internalActions.ts

```diff
@@ -258,7 +258,10 @@ export const setCurrentSandbox: AsyncAction<Sandbox> = async (
   effects.executor.setupExecutor();
 
   effects.fsSync.syncCurrentSandbox();
-  effects.router.updateSandboxUrl(sandbox);
+
+  if (oldSandboxId && oldSandboxId !== state.editor.currentId) {
```

Comment:
No need to check if `oldSandboxId` exists here.

Reply:
Actually we do :) We only want to check if they are different if there actually existed an old id. Basically, we do not want to check if they are different if it is the first sandbox you are opening. It could most certainly be expressed better :)

Reply:
This thing has actually been removed :)